### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by samhita-alla**
image

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
-  Flyte is a <b>workflow automation</b> platform for <b>complex</b>, <b>mission-critical data</b> and <b>ML processes</b> at scale 
-</p>
+  <p align="center">
+   <h3>
+    Flyte is a workflow automation platform for complex, mission-critical data and ML processes at scale 
+   </h3>
+  </p>
 </html>
+
+We at Flyte are constantly striving to make MLOps better by providing off-the-shelf orchestration mechanisms to automate the way you build pipelines. Ours is a **Kubernetes-native** and **type-safe** platform—which we are super proud of! 🥲
```
Comment (kumare3) [inline profile/README.md]: why the cry emoji

File: profile/README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+We at Flyte are constantly striving to make MLOps better by providing off-the-shelf orchestration mechanisms to automate the way you build pipelines. Ours is a **Kubernetes-native** and **type-safe** platform—which we are super proud of! 🥲 
+
+#### 5️⃣ Reasons to Use Flyte
+- Kubernetes-Native Workflow Automation Platform 
+- Ergonomic SDK's in Python, Java & Scala
+- Versioned & Auditable 
+- Reproducible Pipelines
+- Strong Data Typing
+
+Explore our GitHub repo and give us a star ⭐️ if you like what you see 👀. Our [getting started guide](https://docs.flyte.org/en/latest/getting_started.html) should further intrigue you 🧐.
```
Comment (kumare3) [inline profile/README.md]: i think we should say where to star - the flyte repo

File: profile/README.md (hunk 1 above)
Comment (samhita-alla) [reply profile/README.md]: Oh, it's not a cry emoji! It's happy tears. Let me change it.

**Comment by samhita-alla** [comment]
@kumare3 updated README
image

**Review by kumare3 — APPROVED** [review]

**Pull request merged**